packages/app-project/src/screens/ProjectAboutPage/components/AboutSidebar/AboutSidebar.js

```diff
@@ -6,7 +6,7 @@ import { arrayOf, object, string } from 'prop-types'
 const AboutSidebar = ({ aboutNavLinks }) => {
   const router = useRouter()
   const { owner, project } = router.query
-  const baseUrl = `/projects/${owner}/${project}/about`
+  const baseUrl = `/projects/${owner}/${project}/preview/about`
```

Comment:
One non-blocking suggestion because I'm not sure if mobile-width screens are important for this preview, but the same `baseUrl` is used in AboutDropdownNav component so `/preview` may need to be added there too!